Vending & Buyingstore table updates, and buysing store checks (#6718)
rathena/rathena: doc/mapflags.txt

---------------------------------------

*nochat

Disables chatroom creation on a map.

---------------------------------------

*novending

> Disables shop creation on a map by using MC_VENDING skill.

Reviewer: ```suggestion
Disables shop creation on a map from the MC_VENDING skill.
```

---------------------------------------

*nobuyingstore

> Disables shop creation on a map by using ALL_BUYING_STORE skill.

Reviewer: ```suggestion
Disables shop creation on a map from the ALL_BUYING_STORE skill.
```
